pkg/cmd/grafana-cli/commands/install_command.go

```diff
@@ -51,8 +51,8 @@ func validateInput(c utils.CommandLine, pluginFolder string) error {
 }
 
 func (cmd Command) installCommand(c utils.CommandLine) error {
-	pluginFolder := c.PluginDirectory()
-	if err := validateInput(c, pluginFolder); err != nil {
+	pluginsDir := setting.PluginsPath
```

Comment:
Where does `setting.PluginsPath` get set to the same value as the `--pluginsDir` option to grafana-cli?

Reply:
I am realizing an important chunk of this PR is missing: loading value from configuration file